### DISCUSSION TITLE
Show password is not working

### DISCUSSION THREAD

**Discussion by Hancie123** [TomSchimansky/CustomTkinter#597] [Q&A] [answered]
password_txt = customtkinter.CTkEntry(master=frame2, show="*", corner_radius=10, text_font=font1, width=200)
        password_txt.place(x=330, y=250)

        def show_password():
            if password_txt.cget('show') == '*':
                password_txt.config(show='')
            else:
                password_txt.config(show='*')

        i = IntVar()

        password_show = customtkinter.CTkCheckBox(master=frame2, text="Show password", variable=i, command=show_password)
        password_show.place(x=330, y=290)

**Comment by abhaymakes** [comment, marked as answer]
Here is the updated code : 
```python
import customtkinter as c

root = c.CTk()


def show_password():
  if i.get() == 1:
    e.configure(show='')
  else:
    e.configure(show='*')



e = c.CTkEntry(root, show="*")
e.pack()

i = c.IntVar()

password_show = (root, text="Show password", variable=i, command=show_password)
password_show.pack()


root.mainloop()

```

**Reply by Hancie123** [reply]
Thank you so much! The code is working perfectly....
